templates/attachments.tpl

```diff
@@ -43,7 +43,7 @@
                 </td>
               {/if}
               <td>
-                <a href="file/{$problem->name}/{$d.a->name}">
+                <a href="file.php?problemName={$problem->name}&amp;fileName={$d.a->name}">
```

Comment:
This is ok, but in the long run we should make URLs work without .php extensions. For example, images in problem statements can be included with

    !>file/problem_name/image.jpg!